partcad-cli/src/partcad_cli/click/command.py

```
@@ -3,6 +3,7 @@
 import partcad as pc
 import coloredlogs
 import logging
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)